crates/sui-types/src/crypto.rs

```diff
@@ -864,7 +865,7 @@ impl Signer<Signature> for Secp256k1KeyPair {
 #[derive(Clone, Debug, Serialize, Deserialize, JsonSchema, PartialEq, Eq, Hash)]
 pub struct Secp256r1SuiSignature(
     #[schemars(with = "Base64")]
-    #[serde_as(as = "Readable<Base64, Bytes>")]
+    #[serde_as(as = "Readable<ToArray<Base64>, Bytes>")]
```

Comment:
what does this ToArray do?

Reply:
ah i see... this is to use the sized array, why do we need it? (context is im probably going to replace this with a fastcrypto trait to do just this in a later PR)

Reply:
ya we need this because fast crypto's `Base64` only support serde_as Vec<u8>

Reply:
this is used so I don't have to make changes in fast crypto 🙃, we can remove this once serde_as array is supported